app/routes.py

```diff
@@ -1,2 +1,227 @@
-from flask import Blueprint
+from flask import Blueprint, request, jsonify
+from app import db
+from .models.task import Task
+from .models.goal import Goal
+from datetime import datetime
+import requests
+import os
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+#---------------------# HELPER FUNCTIONS #---------------------#
+
+def invalid_input():
+    return jsonify({"details":"Invalid data"}), 400
+
+def post_to_slack(message):
+    path = "https://slack.com/api/chat.postMessage"
+    headers = {"Authorization": f"Bearer {os.environ.get('SLACK_TOKEN')}"}
+    query_params = {"channel": "task-notifications",
+                    "text": message
+                    }
+    requests.post(path, params=query_params, headers=headers)
+
+#---------------------# DECORATORS #---------------------#
+
```

Comment:
Awesome work with Decorators.

If you noticed this file is getting pretty long.  Maybe consider breaking things up into separate files.  Maybe a file for decorators and another for tasks routes and another for goals routes.